client/src/utils.ts

```diff
@@ -139,3 +141,81 @@ export function onVisibilityChange(): void {
     });
   }
 }
+
+function getTopicsParams(url = location.href) {
```

Comment:
This function is getting **all** query params, not just the topic params, so it should probably be named something like "getAllSearchParams"

Reply:
Changed!